molecule/molecule-runtime/src/commonMain/kotlin/app/cash/molecule/GatedFrameClock.kt

```diff
@@ -51,7 +47,7 @@ internal class GatedFrameClock(scope: CoroutineScope) : MonotonicFrameClock {
     }
 
   private fun sendFrame() {
-    clock.sendFrame(start.elapsedNow().inWholeNanoseconds)
+    clock.sendFrame(0L)
```

Comment:
We can add this (or expect/actual platform APIs for now) if someone complains, but I'm pretty sure no one needs/wants it.